frontend/src/components/Navbar.js

```diff
@@ -12,12 +13,58 @@ import MenuItem from "@material-ui/core/MenuItem"
 import AccountCircle from "@material-ui/icons/AccountCircle"
 import NotificationsIcon from "@material-ui/icons/Notifications"
 import Typography from "@material-ui/core/Typography"
+import MenuIcon from "@material-ui/icons/Menu"
 import GsTitle from "./GsTitle"
+import clsx from "clsx"
+
+const NavHeader = observer(({ drawerOpen, handleDrawerOpen, drawerWidth }) => {
+  const useStyles = makeStyles(theme => ({
+    appBar: {
+      transition: theme.transitions.create(["margin", "width"], {
+        easing: theme.transitions.easing.sharp,
+        duration: theme.transitions.duration.leavingScreen,
+      }),
+      "& .navbar__logo": {
```

Comment:
I moved some of the css from the external style sheet into the component.

Reply:
This is good. We are using so little SCSS I would like to remove it if feasible.